### DISCUSSION TITLE
Can anti-alias be improved?

### DISCUSSION THREAD

**Discussion by saint11** [rbreu/beeref#96] [Ideas & General Discussion]
I noticed that at smaller scales the anti-aliasing for images is not very good. I tried to improve this myself, but it seems that this is a problem with the Qt Gui, unless I'm missing something.

I know this alias problem is nothing major, but for a software mostly used for art and by artists I think visual fidelity is important.

Comparison with PureRef for a zoomed out image, BeeRef on the right:
![image](https://github.com/rbreu/beeref/assets/4672703/c823d285-1f4f-417b-9b8c-821d7dc8b186)

**Comment by rbreu** [comment]
Yeah, I'm just relying on the SmoothPixmapTransform rendering that Qt offers, and there's nothing more that can be done as far as I'm aware. The only thing that I'm doing is turning off SmoothPixmapTransform when you are zoomed in past 200% on an image (at which point I'm assuming you want to see individual pixels crisply, e.g. for pixel art), so there's potential for bugs where SmoothPixmapTransform is turned off when it shouldn't.
